checkLdapPwdExpiration.sh

```diff
@@ -62,6 +66,12 @@
 # Configuration
 #====================================================================
 
+#
+# Locale
+# eg: export LC_ALL=en_US.UTF-8
+#
+export LC_ALL=en_US.UTF-8
+
```

Comment:
This shouldn't be necessary, users might want to display the date in their own locale in the email by setting an environment variable. If you need this for in the date parsing or non-user printing, you could temporarily set it:
```
LC_ALL=en_US.UT-8 date [something]
```

Reply:
Since the email text is in english, I tought why not force locale to english by default ?

Reply:
That's a good point, in that case you should put the locale definition and the email body/subject variables closer together so that users have all the internationalization settings in the same place.

Reply:
Agree with @maxbes please move these lines just over MY_MAIL_BODY

Reply:
made modifications in d1bdbc6